.tests/integration/units.tsv

```diff
@@ -1,4 +1,4 @@
-sample	type	run	lane	fastq1	fastq2	adapter
-HD827sonic	N	run1	lane1	fastq/HD827sonic_testing1_R1.fastq.gz	fastq/HD827sonic_testing1_R2.fastq.gz	AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT
-HD827sonic	N	run1	lane2	fastq/HD827sonic_testing2_R1.fastq.gz	fastq/HD827sonic_testing2_R2.fastq.gz	AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT
-HD827sonic	T	run1	lane3	fastq/HD827sonic_testing3_R1.fastq.gz	fastq/HD827sonic_testing3_R2.fastq.gz	AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT
+sample	type	platform	run	lane	fastq1	fastq2	adapter
+HD827sonic	NextSeq	NextSeq	N	run1	lane1	fastq/HD827sonic_testing1_R1.fastq.gz	fastq/HD827sonic_testing1_R2.fastq.gz	AGATCGGAAGAGCACACGTCTGAACTCCAGTCA,AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT
```

Comment:
There are some mistakes in the order of the values 😉

Reply:
😑